files/nrpe/check_cpu

```diff
@@ -100,39 +99,29 @@ val_wcdiff() {
     fi
 }
 
-update_save_file() {
-    if ! grep -m1 '^cpu' /proc/stat > "$save_file" ; then
-	echo Error: Cannot update statistics file.
-	exit $ST_UK
```

Comment:
Here the process statistics read operation + tmp write operation are kind of bundled together. Removing the check for if the former succeeded seems like a regression (since all tmp1/tmp2 might silently go bonkers), but what's your thinking about this. "If we can't read proc, we have bigger problems than tracking cpu usage" ? I *might* buy that, just curious.

Reply:
I have not consider it a check for reading `/proc/stat`, I have only consider it a write check. I don't see that happen, and I consider the worst case scenario to be unimportant that I therefor don't want to implement that check.